vowpalwabbit/multilabel_oaa.cc

```diff
@@ -5,15 +5,18 @@
 #include <cfloat>
 #include "reductions.h"
 #include "vw.h"
+#include "shared_data.h"
 
 #include "io/logger.h"
 
 using namespace VW::config;
+using namespace VW::LEARNER;
```

Comment:
We should avoid usings like this whenever possible. Of course this comes under personal preference but it seems like there are only 3 functions from this namespace used here so in my opinion we should not use this `using`

Reply:
Yea took it out, it was only being used in two places here